fog/view/server/src/server.rs

```diff
@@ -391,6 +391,13 @@ where
     /// Keeps track of which blocks we have fed into the enclave.
     enclave_block_tracker: BlockTracker,
 
+    /// Flag which the db fetcher sets to indicate when it has exhausted it's
+    /// initial work.
+    db_fetcher_readiness_indicator: ReadinessIndicator,
+
+    /// Flag which we set to indicate when the server as a whole is read.
```

Comment:
Nit: I am assuming "ready" was meant here.
```suggestion
    /// Flag which we set to indicate when the server as a whole is ready.
```